pkg/protostore/protostore.go

```diff
@@ -48,9 +48,11 @@ type ProtoStore interface {
 	Update(ctx context.Context, id string, msg proto.Message) (rowsAffected int64, md Metadata, err error)
 	Upsert(ctx context.Context, id string, msg proto.Message) (rowsAffected int64, md Metadata, err error)
 	FindById(ctx context.Context, id string, msg proto.Message, metadata *Metadata) error
+	FindByIdIncludingSoftDeleted(ctx context.Context, id string, msg proto.Message, metadata *Metadata) error
 	FindAll(ctx context.Context, msgs interface{}, pagination *datastore.Pagination) (metadataMap map[string]Metadata, err error)
+	FindAllIncludingSoftDeleted(ctx context.Context, msgs interface{}, pagination *datastore.Pagination) (metadataMap map[string]Metadata, err error)
 	FindAllAsMap(ctx context.Context, msgsMap interface{}, pagination *datastore.Pagination) (metadataMap map[string]Metadata, err error)
-	SoftDeleteById(ctx context.Context, id string, msg proto.Message) (rowsAffected int64, err error)
+	SoftDeleteById(ctx context.Context, id string, msg proto.Message) (rowsAffected int64, md Metadata, err error)
```

Comment:
thank you

Reply:
Hi. Do I need to make any changes in this function?

Reply:
nope